po/perform_translations/translator.py

```diff
@@ -0,0 +1,97 @@
+import re
+import dl_translate as dlt
+import polib
+import os
+
+mt = dlt.TranslationModel()
+
+
+# Variables for translator to edit
+
+print(mt.get_lang_code_map()) 
+
+language = str(input("What is your desired language to translate to? Input the language code: ")) # change to desired language to convert TO
+Reportmsgid = str(input("What is your desired Report Msgid Bugs To? " ))
+POT_Creation_Date = str(input("What is your desired POT Creation Date? "))
+PO_Revision_Date = str(input("What is your desired PO Revision Date? "))
+Last_Translator = str(input("Who do you want as the last translator? "))
+Language_Team = str(input("What is the language team you want the translation to be in? Example: English <myteam@example.com> "))
+
+
+while language not in mt.available_codes():
+  print("Sorry try again! The language code is incorrect")
```

Comment:
You should maybe say that the language is not supported (not all valid languages are supported by dl-translate)